src/app/core/default-values/dynamic-placeholder-value.service.ts

```diff
@@ -52,4 +53,12 @@ export class DynamicPlaceholderValueService extends DefaultValueStrategy {
         break;
     }
   }
+
+  getDefaultValueString(filterConfig: FilterConfig): string {
```

Comment:
I would to suggest the following, simplified method signature:

```suggestion
  /**
   * Get the current value for a given placeholder.
   * @param placeholder The placeholder string to resolve.
   * @returns The value for the given placeholder or `undefined` if not found.
   */
  getPlaceholderValue(placeholder: string): string | undefined {
```


Can you also refactor the `setDefaultValue` above to use this new function? We can replace the switch-statement with a call to this new `getPlaceholderValue` method. (Or maybe actually move the switch statement into the new method, so that it can also resolve other placeholders than CURRENT_USER).